src/histolab/slide.py

```diff
@@ -22,15 +22,17 @@
 """
 
 import math
-import ntpath
-import numpy as np
-import openslide
 import os
-import PIL
 import pathlib
+from typing import Tuple, Union
+
```

Comment:
why this blank line?

Reply:
PEP8 suggestions enforced by Black: https://www.python.org/dev/peps/pep-0008/#imports

```
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

You should put a blank line between each group of imports.
```